brain_games/games/gcd.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+from brain_games.cli import get_data, num_random
+
+RULES = "Find the greatest common divisor of given numbers."
+
+
+def gcd() -> Dict[str, str]:
+    """
+    Forms brain-gcd game round data
+
+    :return: brain-gcd game round data
+    :rtype: Dict[str, str]
+    """
+
+    num1 = num_random(0, 100)
+    num2 = num_random(0, 100)
+
+    while num1 == 0 and num2 == 0:
+        num1 = num_random(0, 100)
+        # num2 = num_random(0, 100)
+
+    if (num1 or num2) == 0 or (num1 or num2) == 1:
+        question = f"{num1} {num2}"
+        correct_answer = max(num1, num2) if (num1 or num2) == 0 else 1
+        return get_data(RULES, question, str(correct_answer))
+
+    question = f"{num1} {num2}"
+
+    if num1 == num2:
+        correct_answer = num1
+
+    elif num1 % num2 == 0 or num2 % num1 == 0:
+        correct_answer = num2 if num1 % num2 == 0 else num1
+
+    else:
+        min_num = num1 if num2 > num1 else num2
+        for d in range(min_num - 1, 0, -1):
+            if num1 % d == 0 and num2 % d == 0:
+                correct_answer = d
+                break
```

Comment:
it's better to make it a separate function: it will generate two numbers, find their gcd and return tuple (first_number, second_number, gcd)